lib/openc_bot/helpers/incremental_search.rb

```diff
@@ -24,12 +24,13 @@ def fetch_gaps
         $stderr.puts "#{Time.now.utc.iso8601} - Starting fetch_gaps"
         current_number = nil 
         gap_count = 0
-        gap_uid = get_var("highest_gap_uid") || sqlite_magic_connection.execute("select min(company_number) from ocdata")[0]["min(company_number)"]
+        save_var(:lowest_company_number, lowest_company_number) if get_var("lowest_company_number").nil?
```

Comment:
@oc-raghav-arneja I think we need to keep the same key both in save_var & get_var
As in save_var you are using key **:lowest_company_number** but in get_var you are accessing it like "lowest_company_number" so I think there should be the same formate